src/main/java/pl/edu/agh/iet/katabank/Bank.java

```diff
@@ -39,4 +40,13 @@ private void checkOperationNotAllowed(Customer customer, Account account, String
             throw new RuntimeException(message);
         }
     }
+
+    public Set<Deposit> getDepositsForCustomer(Customer customer) {
+        return bankProductsRepository.findDepositsForCustomer(customer);
+    }
+
+    public Deposit openDeposit (Customer customer, Account account, BigDecimal depositBalance){
+        checkOperationNotAllowed(customer, account, ERROR_MESSAGE_OPEN_DEPOSIT);
```

Comment:
Please write unit test for this line of code.

Reply:
Added the: "customer cannot open deposit for other customer account" unit test in BankTest